cell/src/main/java/jetbrains/jetpad/cell/text/TextEditingTrait.java

```diff
@@ -181,16 +181,13 @@ public void onPaste(Cell cell, PasteEvent event) {
 
     String text = content.get(ContentKinds.TEXT);
 
-    StringBuilder newText = new StringBuilder();
-    for (int i = 0; i < text.length(); i++) {
-      if (text.charAt(i) != '\n') {
-        newText.append(text.charAt(i));
-      }
+    if (text.contains("\n") || text.contains("\r")) {
```

Comment:
I don't like this solution. It would be nice to specify ContentKinds.MULTILINE_TEXT explicitly.